### PR TITLE
Fix #168 links in messages aren't clickable

### LINKED ISSUE #168: Can't click on links
Clicking on HTTP links in messages is broken.

### PR DESCRIPTION
**Pull request by kenodoggy**
bugfix for links not being clickable, adds autoLink feature including web, phone, map and email

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@kenodoggy, thanks for the pull request! It looks like you haven't yet signed the Dropbox CLA. Please [sign it here](https://opensource.dropbox.com/cla/).

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*
				
@kenodoggy, thanks for signing the CLA!

File: app/src/main/java/com/zulip/android/models/Message.java [hunk 1]
```diff
@@ -25,16 +8,36 @@
...
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.Callable;
+
+import static com.zulip.android.util.CustomHtmlToSpannedConverter.linkifySpanned;
```
Comment (niftynei) [inline app/src/main/java/com/zulip/android/models/Message.java]: If you're only using this in one location, my preference is that the ref for this stays inline (not imported as a static)

Comment (kenodoggy) [inline app/src/main/java/com/zulip/android/models/Message.java]: updated in 48fdf19

**Comment by niftynei** [comment]
This has been merged.  Thanks for the quick turn-around on this!

Just a note: for this project, we ask that commit messages be capitalized.  I went ahead and fixed up these commits to get them merged. 👌

**Pull request closed without merging**